src/components/offering/bio.js

```diff
@@ -11,7 +11,7 @@ const Bio = ({ member }) => {
       <div key={id}>
         <img className='staff-image' src={image[0].url} alt={image[0].alternativeText} />
         <h2 className='h3 staff-name'>{Name}</h2>
-        <a className='linkedin-icon' href={Linkedin}><img src={linkedinIcon} /></a>
+        <a className='linkedin-icon' href={Linkedin}><img src={linkedinIcon} alt=''/></a>
```

Comment:
@jaube05 This is probably out of scope so let's discuss in a different ticket. 

I think we may need visually hidden text for the linkedin icon and a hover state to indicate that it is clickable. I created https://civicactions-home.atlassian.net/browse/HOMER20-480 for this.